utility/index.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require('bcryptjs');
+
+class Utility {
+    async bcryptPass(password) {
+        try {
+            let hashpass;
+            if (password) {
+                var salt = bcrypt.genSaltSync(2);
+                hashpass = await bcrypt.hashSync(password, salt);
+            }
+            return hashpass;
```

Comment:
if password is not defined, then hashpass will return undefined, fix this

Reply:
okk fine sir

Reply:
resolve